Fix home meta title (#60)
rsksmart/devportal: rif/rns/operations/set-btc-address.md

---
layout: rsk
title: Set chain address resolution
---

This operation is used to set a chain address resolution of a domain. If you are using your domain to receive assets, you may use this guide to change the wallet address where you receive payments.

1. Go to [RNS Manager](https://manager.rns.rifos.org)
2. Login
3. Go to admin page
4. If you are using the multi chain resolver, an admin resolution button will appear. Click on it.
  ![set-btc-address-1](/assets/img/rns/set-btc-address-1.png)

>   If you are not using multi-chain resolver find [here](../set-resolver) how to do it.

Reviewer: ```suggestion
  If you are not using multi-chain resolver, follow [these instructions](../set-resolver).
```
